common/djangoapps/track/backends/logger.py

```diff
@@ -33,7 +33,13 @@ def __init__(self, name, **kwargs):
         self.event_logger = logging.getLogger(name)
 
     def send(self, event):
-        event_str = json.dumps(event, cls=DateTimeJSONEncoder)
+        try:
+            event_str = json.dumps(event, cls=DateTimeJSONEncoder)
+        except UnicodeDecodeError:
+            for key in event.iterkeys():
+                if isinstance(event[key], str):
+                    event[key] = event[key].decode('latin1')
```

Comment:
I'm also confused by this -- how is it that there are 'latin1' strings in the first place, and more to the point, why do believe that we should be using latin1 instead of, say, utf-8?  
That said, I'd be more comfortable if we were able to identify the code that putting the wrong data *into* the event in the first place, and fixing that.